actions/create_uplink.py

```diff
@@ -25,16 +24,23 @@ def create_uplink(api_auth, parameters, contexts):
         logging.error(error_string)
         return error_string
 
-    # Get all the sites and check whether there is a site match given city and site type
-    if uplink_name == "":
-        uplink_name = "Uplink";
+    #if uplink_name == "":
+        uplink_name = "Uplink"
```

Comment:
Does this just always make uplinks with the name "Uplink"? If so, you could probably remove the line `uplink_name = parameters["Uplinks"]` near the top of the file.

Reply:
Ah yes, it does always make it with the name 'Uplink'. This was a pretty recent change when i discovered dialogflow is real weird with what it decides is the uplink name, so i ended up with weird names sometimes depending on how i asked it to create one. So i just changed them all to Uplink but kept that in there in case we wanted to make them nameable.